src/app/app.component.ts

```diff
@@ -1,9 +1,52 @@
-import {Component} from '@angular/core';
+import { Component } from '@angular/core';
+import { StateService } from "./state.service";
+import { AngularFire, FirebaseListObservable, AuthProviders, AuthMethods } from 'angularfire2';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
```

Comment:
Firebase stuff shouldn't be happening in the app component. Please create a separate component